### PR TITLE
Rework the target to be a root capability and rename capabilityGrant to capabilityDelegate

### PR DESCRIPTION
**Pull request by cwebber**
These turned out to be somewhat intertwined so done in one pass but across two commits.

### CODE REVIEW

**Comment by cwebber** [comment]
This closes #14 and #13 in one swoop btw.

File: index.html [hunk 1]
```diff
@@ -201,30 +203,28 @@ <h2>OCAP-LD by Example</h2>
...
 
      // Finally we sign this object with cryptographic material from
-     // Alyssa's Car's capabilityGrant field, and using the
-     // capabilityGrant proofPurpose.
+     // Alyssa's Car's capabilityDelegate field, and using the
+     // capabilityDelegate proofPurpose.
      "proof": {
         "type": "RsaSignature2016",
-        "proofPurpose": "capabilityGrant",
+        "proofPurpose": "capabilityDelegate",
```
[the remark this answers is not in the document]
Comment (cwebber) [reply index.html]: I have no problem with that.  I'll update it.

File: index.html [hunk 2]
```diff
@@ -421,6 +421,12 @@ <h2>Terminology</h2>
           (linked data) object.
         </dd>
 
+        <dt>target</dt>
+        <dd>
+          The entity that a capability, upon being invoked, permits the authority
```
[the remark this answers is not in the document]
Comment (cwebber) [reply index.html]: Sounds good, will update.

File: index.html [hunk 3]
```diff
@@ -470,53 +498,79 @@ <h2>Terminology</h2>
...
+          delegate authority to the first non-target entity, we do so using the
+          target's cryptographic authority.
+          If this authority can be used to grant capabilities, it may as well
+          be able to be used to invoke them as well.
+        </p>
+      </div>
+
       <p>
-        Every capability document MUST have:
+        Every capability document, excepting the target, MUST have:
```
[the remark this answers is not in the document]
Comment (cwebber) [reply index.html]: ack

File: index.html [hunk 4]
```diff
@@ -470,53 +498,79 @@ <h2>Terminology</h2>
...
           <!-- TODO: Should this be by any of the authentication material which
                has authority to use this capability, or only the nearest
                invoker on the chain? -->
         </li>
       </ul>
 
       <p>
-        "Top-level" capabilities, which do not have a
-        <code>parentCapability</code> property, MUST have:
+        Every capability document, excepting the target, MAY have:
```
[the remark this answers is not in the document]
Comment (cwebber) [reply index.html]: ack

File: index.html [hunk 5]
```diff
@@ -421,6 +421,11 @@ <h2>Terminology</h2>
           (linked data) object.
         </dd>
 
+        <dt>target</dt>
```
Comment (dlongley) [inline index.html]: Why was this renamed from "invocationTarget" to just "target"? I think it's more likely to conflict with other vocabularies making plain JSON implementations more difficult.

Comment (cwebber) [reply index.html]: There is no invocationTarget or any such target property anymore: since the target is the root capability, we now use parentCapability to point to it.  So this just refers to the concept of a target, not a target property.

Maybe there's a way to make that clearer?  What do you think?
